src/extension/ln/webbtc/index.ts

```diff
@@ -102,43 +110,6 @@ export default class WebBTCProvider {
     action: string,
     args?: Record<string, unknown>
   ): Promise<Record<string, unknown>> {
-    return new Promise((resolve, reject) => {
-      // post the request to the content script. from there it gets passed to the background script and back
-      // in page script can not directly connect to the background script
-      window.postMessage(
-        {
-          application: "LBE",
-          prompt: true,
-          action: `webln/${action}`,
-          scope: "webln",
-          args,
-        },
-        "*" // TODO use origin
-      );
-
-      function handleWindowMessage(messageEvent: MessageEvent) {
-        // check if it is a relevant message
-        // there are some other events happening
-        if (
-          !messageEvent.data ||
-          !messageEvent.data.response ||
-          messageEvent.data.application !== "LBE"
-        ) {
-          return;
-        }
-        if (messageEvent.data.data.error) {
-          reject(new Error(messageEvent.data.data.error));
-        } else {
-          // 1. data: the message data
-          // 2. data: the data passed as data to the message
-          // 3. data: the actual response data
-          resolve(messageEvent.data.data.data);
-        }
-        // For some reason must happen only at the end of this function
-        window.removeEventListener("message", handleWindowMessage);
-      }
-
-      window.addEventListener("message", handleWindowMessage);
-    });
+    return postMessage(action, "webln", args);
```

Comment:
should this be "webbtc?"

Reply:
no, webbtc currently is handled by webln actions internally. that will change with the mnemonic / psbt PR soon, though.